docs/ProductSummaryBuyButton.md

```diff
@@ -1,72 +1,61 @@
-# Product Summary Buy Button
+📢 Use this project, [contribute](https://github.com/vtex-apps/product-summary) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
 
-## Description
+# Product Summary Buy Button
 
 `ProductSummaryBuyButton` is a VTEX Component that renders the buy button.
 This Component can be imported and used by any VTEX App.
```

Comment:
Instead of this sentence, I strongly recommend to add a media here (Buy Button being displayed)